src/validate.js

```diff
@@ -36,6 +37,22 @@ export function selectedColumnsHaveDatatype(state) {
   return selectedColumns.length > 0 && columnTypesOk;
 }
 
+// Check that selected continuous columns only contain numbers.
+// @return {boolean}
+export function continuousColumnsHaveOnlyNumbers(state) {
+  const columns = getSelectedContinuousColumns(state);
+  let allNumbers = true;
+  state.data.forEach(function(row, i) {
```

Comment:
Is `.forEach` a problem in IE11?

Reply:
for `NodeLists`, but I think it's just fine for Arrays.  I'll research a bit to be sure.

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach#Browser_compatibility